interview-questions.md

```diff
@@ -28,25 +30,26 @@ Answer the following questions. First, without external resources. Challenge you
 
 4. What is JSX?
 
-  Your answer:
+  Your answer:JSX is a hybrid between JavaScript and HTMl
+
+  Researched answer: Hybrid between JavaScript and HTML which is viewed by the user in the browser.
 
-  Researched answer:
 
 
 
 5. What is state in React?
 
-  Your answer:
+  Your answer: It's where you store property value that belongs in a component.
 
-  Researched answer:
+  Researched answer: The state of react is when an object where you store property values that belongs to the component. When the state object changes, the component re-renders.
 
 
 
 6. STRETCH: What is hoisting in JavaScript?
 
-  Your answer:
+  Your answer:Hoisting is where a a variable is at the top of a code before execution.
```

Comment:
Pretty close! Its about javascript having to bring the code up to its needed place.